### DISCUSSION TITLE
[ FF7 ] Crash after Mithril Mines

### DISCUSSION THREAD

**Discussion by Zagan66** [julianxhokaxhiu/FFNx#291] [Q&A] [answered]
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**
Game Crashed.

**To Reproduce**

1. Go to 'Mithril Mines'
2. Click on 'After seeing the giant snake dead on the pole, leave screen and game crashes.'
4. See error https://postimg.cc/ZBmvycbq

**Expected behavior**
I expected to enter the Mithril Mines.

**Screenshots**

**GPU (please complete the following information):**

- GPU: EVGA GeForce RTX 2080 SUPER FTW3 ULTRA, OVERCLOCKED, 2.75 Slot Extreme Cool 
Triple + iCX2, 65C Gaming, RGB, Metal Backplate, 08G-P4-3287-KR, 8GB GDDR6

- Driver Version [496.13]

**Additional context**
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/7385874/FFNx.log)
[crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/7385875/crash.zip)

**Comment by julianxhokaxhiu** [comment, marked as answer]
You're likely using 7th and you didn't state it. The crash is happening internally into the game engine and not on FFNx, therefore please approach the relative 7th official support channels.

**Comment by Zagan66** [comment]
Ok, the error told me to report it here so I did.

________________________________
